### PR TITLE
fix: [CDS-80756]: UX Feedback for webhooks UI 

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

- Move the `Webhook` Tabs from top right to toolbar navigation

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/pages/webhooks/Webhooks.tsx [hunk 1]
```diff
@@ -156,8 +164,24 @@ export function Webhooks(): JSX.Element {
...
         breadcrumbs={<NGBreadcrumbs customPathParams={{ module }} />}
         className={css.header}
-        content={<WebhooksTabs />}
       />
+      <Page.SubHeader className={css.subHeader}>
+        <Tabs
+          id="webhookTabs"
+          selectedTabId={selectedTabId}
+          onChange={newTabId => {
+            if (newTabId === WebhookTabIds.ListTab) {
```
Comment (sahithikolichala) [inline src/modules/70-pipeline/pages/webhooks/Webhooks.tsx:173]: this logic seems to be redundant , can we make it common and use it at both places

**Review by sahithikolichala — CHANGES_REQUESTED** [review]
Can make onchange functionality to redirect as common utility and use it at both places

File: src/modules/70-pipeline/pages/webhooks/Webhooks.tsx [hunk 2]
```diff
@@ -156,8 +164,24 @@ export function Webhooks(): JSX.Element {
         }
         breadcrumbs={<NGBreadcrumbs customPathParams={{ module }} />}
         className={css.header}
-        content={<WebhooksTabs />}
       />
+      <Page.SubHeader className={css.subHeader}>
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/pages/webhooks/Webhooks.tsx:160]: Common out this part in to a component and use at both the places.

File: src/modules/70-pipeline/pages/webhooks/Webhooks.tsx (hunk 1 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/pages/webhooks/Webhooks.tsx]: done

**Review by sahithikolichala — APPROVED** [review]

File: src/modules/70-pipeline/pages/webhooks/Webhooks.tsx (hunk 2 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/pages/webhooks/Webhooks.tsx:160]: Left it here on purpose as this defines the page semantics and if something needs to be added in the subHeader later we can do that here

**Review by pankajsingh-harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/webhooks/utils.tsx [hunk 3]
```diff
@@ -44,3 +44,8 @@ export enum STATUS {
 export interface Error {
   message: RBACError
 }
+
+export enum WebhookTabIds {
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/webhooks/utils.tsx:48]: enums should be singular

**Review by rameshx — APPROVED** [review]

File: src/modules/70-pipeline/pages/webhooks/utils.tsx (hunk 3 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/pages/webhooks/utils.tsx:48]: Will fix this in follow ups

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**